### DISCUSSION TITLE
Welcome to Baidu KDD Cup 2022 Discussions!

### DISCUSSION THREAD

**Discussion by zhoujingbo** [PaddlePaddle/PaddleSpatial#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with the participants of Baidu KDD Cup 2022. We hope that you:
  * Ask questions you’re wondering about. 
  * Share ideas.
  * Engage with other Baidu KDD Cup 2022 participants.
  *  We suggest posting the discussion in English for the convenience of all participants over the world.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by moonquakes66** [comment]
What does nan data mean? it haven't collected? Or at some point this turb doesn't work?

**Comment by greatofdream** [comment]
I cannot understand the instruction that Argument Parser is forbidden but the baseline use argument parser. Besides, where is tests/test-1.zip?
https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/README.md?plain=1#L61

**Reply by ramber1836** [reply]
The newest baseline is under PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/tests .

**Comment by hyunjuneS** [comment]
how can i get test data?
